actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -938,7 +938,7 @@ def month_name(number)
         #
         #   year_name(1998) # => 1998
         #
-        # If the <tt>:year_format</tt> option is passed:
+        # If the <tt>year_format: ->(year) { "Heisei #{year - 1988}" }</tt> option is passed:
```

Comment:
Please remove this from here and add to the example

Reply:
I reverted the change.